### PR TITLE
5133 Starting with AI Takeover & Engage

### LINKED ISSUE #5133: [Takeover + landing] Starting with AI - Build
[brief](https://docs.google.com/document/d/1Wk1cowPs55rhNhbFCMPeSP-ynoxVP4hfH7ICg0T9X0E/edit#heading=h.gjdgxs)

### PR DESCRIPTION
**Pull request by solazio**
## Done

Add starting with AI takeover and landing page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/) and [/engage/starting-with-ai](http://0.0.0.0:8001/engage/starting-with-ai)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Make sure it complies with the [design](https://github.com/canonical-web-and-design/www.ubuntu.com/issues/5133#event-2333142188)
- Make sure it complies with the [brief](https://docs.google.com/document/d/1Wk1cowPs55rhNhbFCMPeSP-ynoxVP4hfH7ICg0T9X0E/edit#)

## Issue / Card

Fixes #5133

### CODE REVIEW

**Comment by deadlight** [comment]
The image seems broken:
image

**Comment by solazio** [comment]
@anasereijo or @kwm14 can you please have a look at the image and fix it.

**Comment by kwm14** [comment]
## Design review
Looking good @solazio but the SVG seems to be broken...

### Landing page
- [ ] Laptop icon broken
Screenshot 2019-05-20 at 09 17 52

### Takeover
- [ ] Laptop and IoT icon broken
Screenshot 2019-05-20 at 09 19 13

**Comment by anasereijo** [comment]
@kwm14 
I just checked the SVGs you saved down and they are broken. I think you need to re-save them.

Screenshot 2019-05-21 at 13 57 38
Screenshot 2019-05-21 at 13 58 30

**Comment by kwm14** [comment]
@anasereijo interesting...I got them from the Assets server. Will check again and fix up 👍

**Comment by kwm14** [comment]
@solazio I've checked the SVG and re-exported. Let me know if it fixed 👍 

**Landing page illustration**
https://assets.ubuntu.com/v1/b497ea41-ai-get-started.svg

**Takeover**
https://assets.ubuntu.com/v1/83fd9f87-ai-hero-takeover.svg

**Comment by solazio** [comment]
@kwm14 unfortunately it looks exactly the same... :cry:

**Comment by anasereijo** [comment]
@solazio try these ones:
https://assets.ubuntu.com/v1/f5dfbf9a-ai-hero-takeover.svg
https://assets.ubuntu.com/v1/166a4103-ai-get-started.svg

@kwm14 I opened them on Illustrator and fixed it

**Comment by solazio** [comment]
@anasereijo @kwm14  yeey :tada: it works

**Comment by anasereijo** [comment]
wowooo :exploding_head:

**Comment by kwm14** [comment]
Thanks, @anasereijo Adobe to the rescue 💯 

1_rNd_rHSRI_tbG5f-E6zSVA@2x

**Comment by deadlight** [comment]
@anasereijo @kwm14 um... sorry. The circle around the chip is missing:
image

**Comment by anasereijo** [comment]
@deadlight oh no! 👎 

@solazio hope this one works! https://assets.ubuntu.com/v1/4d7a9aa6-ai-hero-takeover.svg

**Comment by solazio** [comment]
@anasereijo still no circle

**Comment by anasereijo** [comment]
here goes another version! https://assets.ubuntu.com/v1/1512854e-ai-hero-takeover-01.svg

this should be fine now... @solazio is my witness!

**Comment by solazio** [comment]
@kwm14 @deadlight the circle and the computer are in the right place! Please review.

**Comment by kwm14** [comment]
## Design review
- Fixed on the landing page
- Still not visible on takeover

Screenshot 2019-05-21 at 15 51 16

**Comment by kwm14** [comment]
@solazio Yay! Design LGTM 👍

**Comment by caldav** [comment]
LGTM, but blocking until the PDF actually is available for download on the lp.

**Comment by deadlight** [comment]
All good!

File: templates/takeovers/_starting-with-ai-takeover.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
+<section class="p-strip--dark p-takeover js-takeover">
+  <div class="row u-clearfix u-vertically-center">
+    <div class="col-7">
+      <h1>Achieve your AI ambitions quickly, reliably and cost-effectively</h1>
+      <p class="p-heading--four u-sv2">
+        Multi-cloud operations for the full enterprise AI stack
+      </p>
+      <a
+        href="/engage/starting-with-ai?utm_source=takeover&utm_medium=takeover&utm_campaign=CY19_DC_AI_Whitepaper_AIConsulting_Takeover"
```
Comment (caldav) [inline templates/takeovers/_starting-with-ai-takeover.html]:
```suggestion
        href="/engage/starting-with-ai?utm_source=takeover&utm_campaign=CY19_DC_AI_Whitepaper_AIConsulting"
```

File: templates/takeovers/_starting-with-ai-takeover.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    <div class="col-5">
+      <p class="p-takeover__image u-align-text--center">
+        <img
+          src="{{ ASSET_SERVER_URL }}1512854e-ai-hero-takeover-01.svg"
+          alt=""
+        />
+      </p>
+      <p class="u-hide--medium u-hide--large">
+        <a
+          href="/engage/starting-with-ai?utm_source=takeover&utm_medium=takeover&utm_campaign=CY19_DC_AI_Whitepaper_AIConsulting_Takeover"
```
Comment (caldav) [inline templates/takeovers/_starting-with-ai-takeover.html]:
```suggestion
          href="/engage/starting-with-ai?utm_source=takeover&utm_campaign=CY19_DC_AI_Whitepaper_AIConsulting"
```

**Pull request merged**